packages/lesswrong/components/sunshineDashboard/SunshineNewCommentsList.jsx

```diff
@@ -37,6 +38,7 @@ const withListOptions = {
   collection: Comments,
   queryName: 'sunshineNewCommentsListQuery',
   fragmentName: 'SelectCommentsList',
+  enableTotal: true,
```

Comment:
If there's currently a backlog of years-old comments that we need to clear, this could be a performance issue. (If there's no such backlog, or we clear it, then it'll be fine.)